v3/src/components/graph/components/graph.tsx

```diff
@@ -116,10 +129,12 @@ export const Graph = observer((
                      onTreatAttributeAs={handleTreatAttrAs}
           />
 
+          <svg ref={plotAreaSVGRef} className='graph-dot-area'>
```

Comment:
Do we want/need both of these `<svg>`s to have the same `className` (`graph-dot-area`)?

Reply:
Nope. Good catch.